Extension/src/LanguageServer/client.ts

```diff
@@ -620,6 +637,32 @@ class DefaultClient implements Client {
         this.model.tagParserStatus.Value = notificationBody.status;
     }
 
+    private updateInactiveRegions(params: InactiveRegionParams): void {
+        let renderOptions: vscode.DecorationRenderOptions = {
+            light: { color: "rgba(125,125,125,1.0)" },
```

Comment:
nit: personal preference (being a Light user 😄): I built this change and am playing with it.  This is a bit too dark to easily distinguish the inactive regions for me.  Can you bump it up a bit?  175,175,175 looks a bit better to me.

Reply:
Definitely! I don't expect these numbers to survive until the feature is complete